server/typeDefs/system.ts

```diff
@@ -65,6 +65,13 @@ const schema = gql`
     allSystems: [String!]!
   }
   extend type Mutation {
+    """
+    Macro: Systems: Add RnD to Simulator
```

Comment:
See note above about making this generic. Also, be sure to include the word "Report" or "Damage" here.